sandbox/grommet-app/src/pages/layouts/kinds/HPEMarketing/HPEPrivateCloud.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Heading,
+  Image,
+  PageContent,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { LinkNext, User, Clock, Trigger, VirtualStorage } from 'grommet-icons';
+import { ProductHightlight } from './ProductHighlight';
+
+export const HPEPrivateCloud = () => {
+  const breakpoint = useContext(ResponsiveContext);
+
+  console.log(breakpoint);
+
+  const columns = ['xsmall', 'small'].includes(breakpoint)
+    ? ['auto']
+    : breakpoint === 'medium'
+    ? {
+        count: 2,
+        size: 'auto',
+      }
+    : {
+        count: 4,
+        size: 'auto',
+      };
+  const rows = ['auto'];
+  const gap = {
+    xsmall: 'medium',
+    small: 'medium',
+    medium: 'medium',
+    large: 'large',
+    xlarge: 'large',
+  };
+
+  const columnGap = {
+    xsmall: 'large',
+    small: 'large',
+    medium: 'medium',
+    large: 'large',
+    xlarge: 'xlarge',
+  };
+
+  return (
+    <PageContent
+      pad={{ vertical: 'xlarge', horizontal: 'xlarge' }}
+      background={{
+        fill: 'horizontal',
+        color: 'background',
+        dark: true,
+      }}
+      fill="horizontal"
+    >
+      <Box align="center" gap={columnGap[breakpoint]}>
+        {/* their heading is level 1 60px for font 
+        size however this would not be semantically correct */}
+        <Heading level={2} margin="none">
+          Introducing HPE Private Cloud AI
+        </Heading>
+        <Box
+          // these are not the correct sizes from the design
+          width="xlarge"
+          height="medium"
+          background={{
```

Comment:
Instead of doing this as a background image, should we put it inline before the text? Then we can set the width to 100% for the image

Reply:
yes I can try that!

Reply:
done